### PR TITLE
Fix MM53372

### PR DESCRIPTION
**Pull request by larkox**
#### Summary
Fix MM-53372

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-53372

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Review by esarafianou — APPROVED** [review]
LGTM, thanks!

File: webapp/channels/src/components/suggestion/suggestion_list.tsx [hunk 1]
```diff
@@ -105,14 +105,14 @@ export default class SuggestionList extends React.PureComponent<Props> {
     announceLabel() {
         const suggestionReadOut = this.props.ariaLiveRef?.current;
         if (suggestionReadOut) {
-            suggestionReadOut.innerHTML = this.currentLabel as string;
+            suggestionReadOut.textContent = this.currentLabel as string;
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/suggestion/suggestion_list.tsx:108]: is this string casting still needed? since textContent can also be null

**Review by M-ZubairAhmed — APPROVED** [review]

**Review by yasserfaraazkhan — APPROVED** [review]
tested it on Firefox , Mac desktop.
working as expected.

**Pull request merged**